server/src/main/java/org/elasticsearch/snapshots/SnapshotInfo.java

```diff
@@ -534,16 +633,16 @@ public static SnapshotInfo fromXContent(final XContentParser parser) throws IOEx
             uuid = name;
         }
         return new SnapshotInfo(new SnapshotId(name, uuid),
-                                indices,
-                                state,
-                                reason,
-                                version,
-                                startTime,
-                                endTime,
-                                totalShards,
-                                successfulShards,
-                                shardFailures,
-                                includeGlobalState);
+            indices,
```

Comment:
Either format is fine with me. It looks like you aren't actually changing the list, though, right? In that case I'd probably not change the format.